UploadCSV/Program.cs

```diff
@@ -20,75 +20,67 @@ static void Main(string[] args)
+                                    worksheet.Cell(row, 5).Value,
+                                    worksheet.Cell(row, 6).Value,
+                                    worksheet.Cell(row, 7).Value,
+                                    worksheet.Cell(row, 8).Value,
+                                    worksheet.Cell(row, 9).Value,
+                                    worksheet.Cell(row, 10).Value,
+                                    worksheet.Cell(row, 11).ValueAsExcelDisplays,
+                                    worksheet.Cell(row, 12).ValueAsExcelDisplays,
+                                    worksheet.Cell(row, 13).Value,
+                                    worksheet.Cell(row, 14).Value,
+                                    worksheet.Cell(row, 15).Value,
+                                    worksheet.Cell(row, 16).Value,
+                                    worksheet.Cell(row, 17).Value);
 
-                                    ExecuteQueryWithoutResult(connection, insertCommand);
-                                }
-                                catch
-                                {
-                                    String insertCommand = string.Format(
-                                        "INSERT INTO crCashReceipt VALUES('{0}','{1}','{2}','{3}','{4}','{5}','{6}',{7},{8},{9},{10},'{11}','{12}','{13}','{14}','{15}', {16},'{17}')",
-                                        worksheet.Cell(row, 0).Value,
-                                        worksheet.Cell(row, 1).Value,
-                                        worksheet.Cell(row, 2).ValueAsExcelDisplays,
-                                        worksheet.Cell(row, 3).Value,
-                                        worksheet.Cell(row, 4).Value,
-                                        worksheet.Cell(row, 5).Value,
-                                        worksheet.Cell(row, 6).Value,
-                                        worksheet.Cell(row, 7).Value,
-                                        worksheet.Cell(row, 8).Value,
-                                        worksheet.Cell(row, 9).Value,
-                                        worksheet.Cell(row, 10).Value,
-                                        worksheet.Cell(row, 11).ValueAsExcelDisplays,
-                                        worksheet.Cell(row, 12).ValueAsExcelDisplays,
-                                        worksheet.Cell(row, 13).Value,
-                                        worksheet.Cell(row, 14).Value,
-                                        worksheet.Cell(row, 15).Value,
-                                        0, // because cFreight is null or empty
-                                        worksheet.Cell(row, 17).Value);
-
-                                    ExecuteQueryWithoutResult(connection, insertCommand);
-                                }
+                                ExecuteQueryWithoutResult(connection, insertCommand);
                             }
+                            catch
```

Comment:
Bakit may logic sa loob ng catch dito? Tapos same lang sa try, dapat catch mo exception dito

Reply:
sa DB, may table na 'cFreight' ay [decimal](18, 0) NULL.
so kapag blank ang cell sa CSV file, nageerror. 
ang catch niya is same query, pero yung 'row[16]' will be set as '0'

Reply:
Lagay mo na lang yung logic dun sa try. Dun mo gawin yung null check. cFreight ?? 0